src/status_im/navigation/screens.cljs

```diff
@@ -1217,6 +1218,20 @@
      :options   {:insets {:top? true :bottom? true}}
      :component backup-recovery-phrase/view}
 
+    {:name      :screen/backup-recovery-phrase-on-shell
```

Comment:
what is on-shell ?

Reply:
The shell version of the flow, we have 3 versions:
- The one for regular screens (wallet)
- The dark one you used
- The shell version for profile settings